cryptofeed/backends/redis.py

```diff
@@ -37,135 +41,265 @@ def __init__(self, host='127.0.0.1', port=6379, socket=None, key=None, numeric_t
+        key = f"{self.key}:{feed}:{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
 
 
 class CandlesRedis(RedisZSetCallback, BackendCandlesCallback):
     default_key = 'candles'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        key = f"{self.key}:{feed}:{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
 
 
 class CandlesStream(RedisStreamCallback, BackendCandlesCallback):
-    default_key = 'candles'
+    default_key = 'candles_stream'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        key = f"{self.key}:{feed}:{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
+
+class PositionsLatestRedis(RedisHashCallback, BackendPositionsCallback):
+    default_key = 'positions_latest'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        account = data['account']
+        key = f"{self.key}:{feed}:{account}:{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
+
+
+class PositionsStream(RedisStreamCallback, BackendPositionsCallback):
+    default_key = 'positions_stream'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        account = data['account']
+        key = f"{self.key}:{feed}:{account}:{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
+
+
+class BalancesLatestRedis(RedisHashCallback, BackendBalancesCallback):
+    default_key = 'balances_latest'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        account = data['account']
+        key = f"{self.key}:{feed}:{account}"
+        if symbol is not None:
+            key += f":{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
+
+
+class BalancesStream(RedisStreamCallback, BackendBalancesCallback):
+    default_key = 'balances_stream'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        account = data['account']
+        key = f"{self.key}:{feed}:{account}"
+        if symbol is not None:
+            key += f":{symbol}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
+
+
+class OrderInfoLatestRedis(RedisHashCallback, BackendOrderInfoCallback):
+    default_key = 'order_info_latest'
+    async def write(self, feed: str, symbol: str, timestamp: float, receipt_timestamp: float, data: dict):
+        order_id = data['order_id']
+        account = data['account']
+        key = f"{self.key}:{feed}:{account}:{symbol}:{order_id}"
+        await super().write(feed, symbol, timestamp, receipt_timestamp, data, key)
```

Comment:
new line :)

Reply:
Sorry, I didn't get it. Should I add new line before 302?